data/data/bootstrap/baremetal/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender

```diff
@@ -6,13 +6,26 @@ case "$STATUS" in
     logger -s "NM local-dns-prepender triggered by ${1} ${2}."
     DNS_IP="127.0.0.1"
     set +e
-    logger -s "NM local-dns-prepender: Checking if local DNS IP is the first entry in resolv.conf"
-    if grep nameserver /etc/resolv.conf | head -n 1 | grep -q "$DNS_IP" ; then
-        logger -s "NM local-dns-prepender: local DNS IP already is the first entry in resolv.conf"
-        exit 0
+    if systemctl -q is-enabled systemd-resolved; then
+        >&2 echo "NM resolv-prepender: Setting up systemd-resolved for local DNS"
```

Comment:
Is there a reason that we are using `>&2 echo` instead of `logger -s`, which is used elsewhere in this file? Is it just because that is what the code that we are copying does?

Reply:
Reworked to use `logger -s` (it was copied verbatim from MCO)